backend/src/main/java/org/example/app/task/logic/UcSaveTaskItem.java

```diff
@@ -32,11 +32,11 @@ public class UcSaveTaskItem {
    * @return the {@link TaskItemEntity#getId() primary key} of the saved {@link TaskItemEntity}.
    */
   // @RolesAllowed(ApplicationAccessControlConfig.PERMISSION_SAVE_TASK_ITEM)
-  public Long save(TaskItemEto item) {
+  public TaskItemEntity save(TaskItemEto item) {
```

Comment:
Returning an Entity from a use case is not good practice. We'll have a talk about this in the training